WiseBuy/Managers/APIManager.m

```diff
@@ -6,243 +6,34 @@
 //
 
 #import "APIManager.h"
-#import "Parse/Parse.h"
-#import "Deal.h"
-#import "Item.h"
-#import "Foundation/Foundation.h"
-#import "DatabaseManager.h"
 
 @implementation APIManager
```

Comment:
As mentioned in your RFC, I think there is a bit more work you can do here to make the API more scalable. This refactor helps improve the readability of the code by separating out the code for each of the three APIs, but you can continue to improve the API by making it easy to scale the APIManager to support new APIs with minimal changes to the APIManager itself